sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/logging/KalixJoranConfigurator.scala

```diff
@@ -73,7 +73,9 @@ class KalixJoranConfigurator extends DefaultJoranConfigurator {
       status
 
     } else {
-      ExecutionStatus.INVOKE_NEXT_IF_ANY
```

Comment:
If I remember correctly, it will load the next configurator if it detects one. But it might be that it doesn't detect the default one because we are overwriting it.